documentation.md

```diff
@@ -29,6 +29,7 @@ This page attempts to track the various relevant documentation that exists for R
 * [How to ensure Large File Support for tools using the rpm API](devel_doc/large_files.html)
 
 ## RPM Language Bindings
+* [RPM Python examples](python.html) - A showcase of real-life Python RPM examples
```

Comment:
I couldn't figure out how to build the website, so I don't know if the link is correct. IMHO it should be.